app/screens/thread/thread_post_list/thread_post_list.tsx

```diff
@@ -95,6 +111,7 @@ const ThreadPostList = ({
             rootId={rootPost.id}
             shouldShowJoinLeaveMessages={false}
             showMoreMessages={isCRTEnabled}
+            header={header}
```

Comment:
if the thread is being fetched, perhaps we need to prevent the pull to refresh functionality ?